packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -657,16 +658,33 @@ export class PeerManager {
    */
   private onLibp2pPeerDisconnect = (evt: CustomEvent<Connection>): void => {
     const {direction, status, remotePeer} = evt.detail;
+    const peerIdStr = remotePeer.toString();
+
+    let logMessage = "A peer disconnected us";
+    const logContext: Record<string, string | number> = {
+      peerId: prettyPrintPeerIdStr(peerIdStr),
+      direction,
+      status,
+    };
+    if (direction === "inbound") {
+      // prevent automatic/immediate reconnects
+      const coolDownMin = this.peerRpcScores.applyReconnectionCoolDown(
+        peerIdStr,
+        GoodByeReasonCode.INBOUND_DISCONNECT_NO_GOODBYE
```

Comment:
even if the client sent a goodbye to us, we still reach this code in the end, so change it to 
```suggestion
        GoodByeReasonCode.INBOUND_DISCONNECT
```
drop some comments for this context that some clients don't sent goodbye to us, like Nimbus

Reply:
updated